packages/yarnpkg-core/sources/Manifest.ts

```diff
@@ -427,6 +433,8 @@ export class Manifest {
           errors.push(new Error(`Invalid built meta field for '${pattern}'`));
           continue;
         }
+        if (typeof built !== `undefined`)
+          topLevelWorkspaceFields.add(`dependenciesMeta ➤ ${pattern} ➤ built`);
```

Comment:
It's really unfortunate that we can't use colors in here, but it feels wrong to pass `configuration` as part of an option bag parameter to `Manifest.prototype.load`.